src/Altinn.Notifications.Core/Configuration/KafkaSettings.cs

```diff
@@ -15,6 +15,11 @@ public class KafkaSettings
     /// </summary>
     public string PastDueOrdersTopicName { get; set; } = string.Empty;
 
+    /// <summary>
+    /// The name of the past due orders retry1 topic
+    /// </summary>
+    public string PastDueOrdersTopicNameRetryFirst { get; set; } = string.Empty;
```

Comment:
would PastDueOrdersTopicNameRetry1 be easier to read? If we eventually also get retry2 and 3?